run_tests

```diff
@@ -2,3 +2,4 @@
 
 vagrant ssh-config > .vagrant/ssh-config
 py.test -vv --hosts=quadlet --sudo --ssh-config=.vagrant/ssh-config tests/*_test.py
+py.test -vv --hosts=client --sudo --ssh-config=.vagrant/ssh-config tests/foreman_client.py
```

Comment:
This made me wonder if we could somehow tag hosts, but I don't think we can.

Perhaps a better design is to introduce our own fixtures for certain roles so we can run a single pytest and not accidentally do the wrong stuff?

https://testinfra.readthedocs.io/en/latest/examples.html#test-docker-images also has an interesting pattern where it spins up containers. https://github.com/theforeman/smoker/blob/master/tests/test_content_client.py also has some some code to spin up clients, but now I wonder if we even run that given it mentions katello-agent a lot.

Reply:
So I ended up creating custom fixtures calling `testinfra.get_host`, which seems to work fine.

And no, we don't run the content client tests in smoker today.